org.keshavnrj.wonderwall.yaml

```diff
@@ -0,0 +1,46 @@
+app-id: org.keshavnrj.wonderwall
+base: io.qt.qtwebkit.BaseApp
+base-version: '5.11'
+runtime: org.kde.Platform
+runtime-version: '5.11'
```

Comment:
This runtime version is obsolete. 5.15 is the current one.

Reply:
I thought QtWebkit module may not be available in Latest versions. But looks like there is base app for 5.15 available. I will use 5.15
Thanks.